packages/utils/src/fetch.ts

```diff
@@ -0,0 +1,157 @@
+/**
+ * Native fetch with transparent and consistent error handling
+ *
+ * [MDN Reference](https://developer.mozilla.org/docs/Web/API/fetch)
+ */
+async function wrappedFetch(url: string | URL, init?: RequestInit): Promise<Response> {
```

Comment:
this should not show up as newly added code as the implementation is just moved from api to utils package, you need to delete the file from `packages/api` so git correctly detects that it was moved, also make sure to move the corresponding unit tests

Reply:
Will do! 
Thank you.